Add extender API that return select config (#299)
zowe/cics-for-zowe-client: packages/sdk/CHANGELOG.md

All notable changes to the IBM® CICS® Plug-in for Zowe CLI will be documented in this file.

## Recent Changes

> - Enhancement: Added a some missing resource constants. [#299](https://github.com/zowe/cics-for-zowe-client/pull/299)

Reviewer: ```suggestion
- Enhancement: Added pipeline and webservice resource constants. [#299](https://github.com/zowe/cics-for-zowe-client/pull/299)
```
